.github/workflows/kotlin-api-client-publish.yaml

```diff
@@ -0,0 +1,48 @@
+name: Build kotlin artifacts and publish to github packages
+
+on:
+  push:
+    branches: [main, dev]
+
+  pull_request:
+    branches: [main, dev]
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
```

Comment:
@jagpreetsinghsasan Please lock it down to the same version of the checkout action that is being used by all the other actions as well. See https://github.com/hyperledger/cactus/pull/2188